lib/active_admin/batch_actions/controller.rb

```diff
@@ -5,9 +5,10 @@ module Controller
       # Controller action that is called when submitting the batch action form
       def batch_action
         if action_present?
-          selection =            params[:collection_selection] ||  []
-          inputs    = JSON.parse params[:batch_action_inputs]  || '{}'
-          inputs    = inputs.with_indifferent_access.slice *instance_exec(&current_batch_action.inputs).keys
+          selection  =            params[:collection_selection] ||  []
+          inputs     = JSON.parse params[:batch_action_inputs]  || '{}'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.